synchronization/src/test/java/de/cyface/synchronization/MeasurementSerializerTest.java

```diff
@@ -100,20 +102,26 @@
      */
     @Mock
     private Cursor geoLocationsCursor;
-
     @Mock
     private FileAccessLayer mockedFileAccessLayer;
-
+    @Mock
+    private File mockedAccelerationFile;
+    @Mock
+    private File mockedRotationFile;
+    @Mock
+    private File mockedDirectionFile;
     private MeasurementSerializer oocut;
-
-    private final static long SERIALIZED_SIZE = BYTES_IN_HEADER + 3L * BYTES_IN_ONE_GEO_LOCATION_ENTRY
-            + 3L * 3L * BYTES_IN_ONE_POINT_3D_ENTRY;
-
-    private final static int SAMPLE_POINT_3DS = 3;
```

Comment:
I did not like that we generate the same amount of AP/RP/DP as we might not catch all bugs, e.g. referencing RP instead of AP somewhere in the code. With different sensor point type counts in the sample sets we should notice such typos with those tests.